components/clarity-repl/src/repl/session.rs

```diff
@@ -60,25 +103,21 @@ pub struct Session {
 
 impl Session {
     pub fn new(settings: SessionSettings) -> Self {
-        let tx_sender = {
-            let address = match settings.initial_deployer {
-                Some(ref entry) => entry.address.clone(),
-                None => format!("{}", StacksAddress::burn_address(false)),
-            };
-            PrincipalData::parse_standard_principal(&address)
-                .expect("Unable to parse deployer's address")
-        };
+        let mut interpreter = ClarityInterpreter::new(
+            settings.get_default_sender(),
+            settings.repl_settings.clone(),
+            settings.cache_location.clone(),
+        );
+
+        set_up_accounts(&settings.initial_accounts, &mut interpreter);
+        let boot_contracts = deploy_boot_contracts(&settings, &mut interpreter);
```

Comment:
The main change of this PR.
Directly set up genesis accounts and boot_contracts here